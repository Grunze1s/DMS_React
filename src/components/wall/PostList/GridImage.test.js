import React from 'react';
import GridImage from './GridImage';
import {
    configure,
    shallow
} from 'enzyme';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
configure({
    adapter: new EnzymeAdapter
})


const setup = (props = {}, state = null) => {
    return (shallow( < GridImage {...props}  />)
    )
}
const dummy_data={
    mediaList:[{image:2},{image:3},{image:4}],
    handleToggle:jest.fn()
}
describe('<GridImage />', () => {
  

it("GridImage render", () => { 
        const wrapper= setup({...dummy_data});
        expect(wrapper).toMatchSnapshot();
  
})
})

