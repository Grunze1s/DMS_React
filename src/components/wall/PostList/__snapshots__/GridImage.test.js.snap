// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridImage /> GridImage render 1`] = `ShallowWrapper {}`;
